temporal/t.select/t.select.html

```diff
@@ -374,9 +374,3 @@ <h2>REFERENCES</h2>
 <h2>AUTHORS</h2>
 
 Thomas Leppelt, S&ouml;ren Gebbert, Th&uuml;nen Institute of Climate-Smart Agriculture
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
-
```

Comment:
Line 116 unnecessary one empty line. 

`<h2>REFERENCES</h2>` section should be before `<h2>SEE ALSO</h2>` section.

Reply:
Line 7 missing one empty line.